File: 이수창/20230512/완주하지 못한 선수.js
function solution(participant, completion) {
    var answer = '';
    participant.sort()
    completion.sort()
    let i=0;
    while(true)
    {
        if(participant[i]!=completion[i])
        {
           answer=participant[i]
           break;
        }
        i++
    }
    return answer;
}
// 사전순서로 completion과 participant 바꾼다음
// completion과 participant가 맞지 않으면 바로 해당 참가자 리턴

File: 서예진A/20230512/완주하지못한선수.cpp
#include <string>
#include <vector>
#include <iostream>
#include <algorithm>

using namespace std;

string solution(vector<string> participant, vector<string> completion) {
    string answer = "";
    
    sort(participant.begin(),participant.end()); //알파벳 순 정렬
    sort(completion.begin(),completion.end()); //알파벳 순 정렬

    int i=0;
    for(i; i<completion.size(); i++)
    {
        if(participant[i]!=completion[i]) //정렬된 배열끼리 비교했을 때 같은 위치에 없는 경우 완주자X
            break;
    }
    
    answer=participant[i];
        
    return answer;
}